### DISCUSSION TITLE
學校提供的免費5TB雲端硬碟

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS4719-C/Python-Community-GS4719-C#511] [General]
教育版Google Workspace提供的免費無限容量雲端硬碟到今年6月就要取消了，學校的Google Workspace目前也不再讓新申請的帳號使用雲端硬碟的功能([相關公告](https://ncu.edu.tw/GoogleApps/news/storage-policy))。  
但是學校其實還提供了微軟Onedrive的5TB雲端空間，只要到[學校Office365申請頁面](https://ncu.edu.tw/office365/login) 就可以申請帳號囉！
申請完後就可以看到帳戶中有5TB的空間可用了～
<img width="853" alt="Screen Shot 2022-03-23 at 15 26 48" src="https://user-images.githubusercontent.com/32983347/159645810-66ad0ccb-fb62-4ae7-9752-ce70077617c0.png">
**要注意Office365帳號限在學學生使用，畢業後學校可能就會刪除帳號了**

**Comment by ThomasMeng0515** [comment]
感謝分享，之前還在想Google的不再免費提供怎麼辦...

**Comment by yucheng1028** [comment]
不知道在上面亂存東西會不會怎樣，之前有高中同學被網管約談過

**Reply by gnsJhenJie** [reply]
理論上不要拿雲端硬碟來公開分享或儲存違反Google或微軟使用條款的檔案（像盜版影片、軟體等）是不會被學校關切的

**Comment by yi-1111** [comment]
快容量不足的我，看到了救命稻草，感謝

**Comment by gnsJhenJie** [comment]
關於Google Workspace的部分學校有在努力和Google談價錢(他真的很貴)，大家還是可以抱著一點希望XD

**Comment by Lai27330** [comment]
感謝分享，這樣系上新聞部為各個活動留下的精彩照片就能更方便的儲存下來供大家下載了，而且手機中圖片庫裡太多照片也會影響到資料備份的空間

**Comment by 11timyyy** [comment]
謝謝分享，5TB還算蠻多的~

**Comment by dayu1004** [comment]
太好了 前幾天才在擔心如果學校的雲端不能使用的話要怎麼辦 如果未來真的有5TB可以使用的話 真的會給學生帶來很大的方便性！

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，5TB可以增加學生的方便性。

**Comment by hapiwangy** [comment]
雖然失去了無限大的存放空間，但5TB對於學生來說應該也夠用了。

**Comment by sean1325418** [comment]
只要小心畢業後要記得備份到其他地方，不然重要資料被刪掉就完了。

**Comment by HuanYuuuuuu** [comment]
最近剛好google雲端的15GB的容量要用完了
正在考慮之後還有哪些地方可以存放。
太感謝了

**Comment by PoooYen** [comment]
5TB真的太香了啦><，感謝分享

**Comment by xkllkx** [comment]
謝謝分享，硬碟備分有救啦。

**Comment by Yuhsin000** [comment]
5TB讚，印象中家用版Office 365一人只有1TB

**Comment by henrywen547** [comment]
5TB真的太香了
真的要謝謝你提醒我還有這個儲存空間!

**Comment by 110409538** [comment]
5TB拿來儲存課業相關的資料也不錯

**Comment by ghost** [comment]
雖然這資訊對我來說有點晚 但還是感謝分享QQ

**Comment by bloodtrail0511** [comment]
每次都被提醒雲端空間不足，感謝分享讓我的資料有地方放

**Comment by yucheng1028** [comment]
謝謝推薦

**Comment by Hchejxnc** [comment]
最近容量剛好用完，感謝分享~

**Comment by d26488399** [comment]
謝謝分享，原來還有5TB可已用

**Comment by leon10456** [comment]
感謝分享 硬碟東西快放不下了

**Comment by HuanYuuuuuu** [comment]
感謝分享 我的google雲端硬碟已經滿了

**Comment by minniewei** [comment]
感謝大大讓我知道這東東的存在！！

**Comment by Rosancl** [comment]
感謝分享，甚至是看到這篇文才知道之後學校google雲端要關了哈哈
